RELEASE_PROCEDURE.md

```diff
@@ -19,6 +19,7 @@
        - Version bumping: Change version from “X.X.Xb” to “X.X.X”. E.g. version=”0.2.0b” to version=”0.2.0”
          - `stellargraph/version.py`
          - `meta.yaml`
+       - Update expected versions of demo notebooks: `scripts/format_notebooks.py -v -o demos/`
```

Comment:
Since this is written down and will be copy pasted rather than retyped (I imagine), I think it makes sense to spell out the flags for clarity, rather than keep them short for writer-convenience:

```suggestion
       - Update expected versions of demo notebooks: `scripts/format_notebooks.py --check_version --overwrite demos/`
```